src/users/users.service.ts

```diff
@@ -37,4 +38,34 @@ export class UsersService {
 
     }
 
+    async createUser(userInfo : UsersCreateBodyDto){
+      if(process.env.ENVIRONMENT == "production"){
```

Comment:
Vamos tentar recaptular a lógica aqui.

Se o ambiente for PRODUCTION devemos cadastrar na threefold. Mas porquê esse IF está antes da verificação do username no nosso banco? O que faz mais sentido para mim é que isso estivesse na linha 55.